src/components/views/rooms/RoomList.tsx

```diff
@@ -594,6 +601,7 @@ export default class RoomList extends React.PureComponent<IProps, IState> {
                 (this.props.activeSpace === MetaSpace.Favourites && orderedTagId !== DefaultTagID.Favourite) ||
                 (this.props.activeSpace === MetaSpace.People && orderedTagId !== DefaultTagID.DM) ||
                 (this.props.activeSpace === MetaSpace.Orphans && orderedTagId === DefaultTagID.DM) ||
+                (this.props.activeSpace === MetaSpace.VideoRooms && orderedTagId === DefaultTagID.DM) ||
```

Comment:
Shouldn't this be `DefaultTagID.Conference`

Reply:
My understanding is: if the active space is the videoRooms space and the current section/tag we are processing (orderedTagId) is the DM tag/lists-ection we set always visible to false -> hence we do not show it.
If I remove this line the videoRooms include DM's (Even dm's that are not video rooms. Which we did not want) A video room with two participant is shown since they should not be tagged as DM's in the account settings except is someone does so manually.